conans/test/unittests/client/generators/virtualenv_test.py

```diff
@@ -0,0 +1,19 @@
+import platform
+import unittest
+
+from conans import ConanFile, Settings
+from conans.client.generators.virtualenv import VirtualEnvGenerator
+from conans.model.env_info import EnvValues
+from conans.test.utils.tools import TestBufferConanOutput
+
+
+@unittest.skipUnless(platform.system() == "Windows", "Test both .sh and .bat files")
```

Comment:
why not mock `platform.system()`?

Reply:
I have moved this check inside the test so it is not skipped when out of windows